caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -1057,8 +1080,21 @@ def save(self,filename):
         else:
             raise Exception("Unsupported file extension")
 
-
     def mc_next(self, t, frame):
+        """
+        Perform online motion correction on the next frame
+
+        Args:
+            t: int
```

Comment:
While you're here, because these types are so simple, mind dropping in the python typing into the prototype for mc_next too? If anything, that's more important than the types being in the comments.

Reply:
will do, I find them more readable in the docstring than in the method signature when there are 5+ arguments

Reply:
Fair, although in such cases it can make sense to split the args out over multiple lines. It's increasingly important to use types in prototypes because IDEs and tools like mypy and pydoc can use types for intelligent things.

Reply:
You could also mark the return type of the function with a type annotation

Reply:
Yup I always put them in the signature for that reason too! I meant it's easier to read when the type annotations are in the docstring because then it will render in autodocs, easier to read when it's in the docs because sphinx always squishes together the function signature into one unreadable blob, example: https://caiman.readthedocs.io/en/master/core_functions.html#cnmf